sdk/runtime/runenv.go

```diff
@@ -301,6 +301,21 @@ func (re *RunParams) IntParam(name string) int {
 	return i
 }
 
+// FloatParam returns a float64 parameter, or -1.0 if the parameter is not set or
+// the conversion failed. It panics on error.
+func (re *RunEnv) FloatParam(name string) float64 {
+	v, ok := re.TestInstanceParams[name]
+	if !ok {
+		return -1.0
+	}
+
+	f, err := strconv.ParseFloat(v, 32)
+	if err != nil {
+		panic(err)
+	}
+	return f
+}
+
```

Comment:
@raulk @nonsense a heads up that this was added in. I'm not currently using it, but every so often I think "maybe I want a float parameter"... happy to remove it if nobody wants it.